dev-data/hero-page.data.ts

```diff
@@ -0,0 +1,23 @@
+export const heroPageData = {
+  school: {
+    mainTitle: 'RS School',
+    widgetTitle: 'Connecting people, growing together, having fun',
+    subTitle: ['Free courses. High motivation'],
+    heroImageSrc: '',
+    imageAltText: '',
+  },
+  courses: {
+    mainTitle: 'Our Courses',
+    widgetTitle: 'Journey to full stack mastery',
+    subTitle: ['Community driven. 100% free of charge'],
+    heroImageSrc: 'src/shared/assets/mentor-with-his-students.webp',
```

Comment:
In case the image name ever changes, we won't see any compilation error. To fix this we could use something like this: ```import coursesHeroImg from '@/shared/assets/mentor-with-his-students.webp';```

Reply:
Done